src/constants/menu-collection.js

```diff
@@ -1,7 +1,6 @@
 export const SIDEBAR_MENUS = [
     {name: "Dashboard", icon: "dashboard", url: "/dashboard"},
     {name: "Uptime", icon: "arrow circle up", url: "/uptime"},
-    {name: "Monitor", icon: "clock", url: "/monitor"},
```

Comment:
`Monitor` is removed ?

Reply:
Yes, as we are not sure what data to show in that tab. I have removed the `Monitor` tab.